packages/docs-reanimated/docs/debugging/accurate-call-stacks.mdx

```diff
@@ -29,13 +36,15 @@ const {
   wrapWithReanimatedMetroConfig,
 } = require('react-native-reanimated/metro-config');
 
-const config = {
-  // Your existing Metro configuration options
-};
-
-module.exports = wrapWithReanimatedMetroConfig(config);
```

Comment:
This also was confusing for some people who copied code from this example with the empty `config` and wondered why RN app doesn't start. I don't know how to best represent this in code, so I just removed `config` so that people can think for a while what this config should be before starting the app.

I think that the best way to represent this would be to show something similar to git diff but I don't know if we have this functionality already added in docs.